### PR TITLE
Feature add playbook to the website

### PR DESCRIPTION
**Pull request by marsolekk**

### CODE REVIEW

File: company_website/static/how_we_work/how_we_work.sass [hunk 1]
```diff
@@ -92,6 +92,64 @@
...
 
 
+.how-we-work-playbook-header
+    .section-header
+        $description-total-height: 140px
+        $description-content-height: 100px
+        $description-text-offset: 16px
+        $description-vertical-padding: calc(#{$description-total-height} - #{$description-content-height} - #{$description-text-offset})
+        $description-background-color: $blue-color
+        $description-text-color: $orange-color
```
Comment (kbeker) [inline company_website/static/how_we_work/how_we_work.sass:102]: Such things shouldn't be included here. It should be in the top of a file. It is better to defined such variables on the top of a file

**Review by kbeker — COMMENTED** [review]
One thing which bothers me is that font size of text which is added there is a lot bigger than in other places. Is it done on purpose? Was it consulted with Karolina?

**Comment by marsolekk** [comment]
> One thing which bothers me is that font size of text which is added there is a lot bigger than in other places. Is it done on purpose? Was it consulted with Karolina?

Yes, all style connected things were consulted with Karolina. We decided to use bigger font, because we want to optically extend subsection. I left normal size in mobile view because bigger looks terrible and cause probems with fitting content to width

File: company_website/static/how_we_work/how_we_work.sass (hunk 1 above)
Comment (marsolekk) [reply company_website/static/how_we_work/how_we_work.sass]: Done

**Review by kbeker — APPROVED** [review]

**Pull request merged**